make.conf

```diff
@@ -82,8 +85,12 @@ QEMU_CMD_CORE = \
 	  \
 	  -virtfs local,path=fs_virtfs,mount_tag=virtfs0,security_model=none \
 	  \
-	  -drive file=fat:rw:fs_esp,format=raw,media=disk \
+	  -drive file=fat:ro:fs_esp,format=raw,media=disk,if=none,id=fatdisk,readonly=on \
+	  -device driver=virtio-blk,drive=fatdisk \
 	  \
-	  -device virtio-net,netdev=nic -netdev user,id=nic
+	  -object "rng-random,filename=/dev/urandom,id=rng0" \
+	  -device "virtio-rng-pci,rng=rng0" \
+	  -nodefaults \
+	  $(NOGUI)
```

Comment:
You've -nodefaults in the QEMU_CMD_CORE and in NOGUI

Reply:
I meant to pull that out of NOGUI.  But where do you think it belongs?

Reply:
> I meant to pull that out of NOGUI. But where do you think it belongs?

QEMU_CMD_CORE